php/utils/class-sqlite-db.php

```diff
@@ -248,7 +248,7 @@ public static function update( $data, $where, $table_name = '', $ee_db = '' ) {
 			$table_name = self::$ee_site_table_name;
 		}
 		$is_table_exist = self::is_site_table_exist( $ee_db );
-		if ( $is_table_exist ) {
+		if ( $is_table_exist && !empty( $fields ) ) {
```

Comment:
@bnap00 Why you are checking `$fields` variable as its not define before this if condition.  I think you are trying to check `$data` variable. :)